src/main/java/dk/kb/datahandler/facade/DsDatahandlerFacade.java

```diff
@@ -471,6 +475,7 @@ public static long updateManifestationForRecords(String origin, Long mTimeFrom)
         // Create a custom ForkJoinPool with configured amount of threads
         int numberOfThreads = ServiceConfig.getPreservicaThreads();
         ForkJoinPool customThreadPool = new ForkJoinPool(numberOfThreads);
+        log.info("");
```

Comment:
Can this log-line be removed?

Reply:
Nice catch. I simply forgot to write the text that i wanted to log. I've updated with an actual message.